litani

```diff
@@ -371,9 +388,9 @@ def positive_int(arg):
     except ValueError as e:
         raise argparse.ArgumentTypeError(
             "Timeout '%s' must be an int" % arg) from e
-    if ret <= 0:
+    if ret < 0:
```

Comment:
Technically, we could rename this to `non_negative_int`.

Reply:
Done in 0149ddd